src/main.tsx

```diff
@@ -17,16 +17,32 @@ import {
   DefaultChain,
   DefaultRpcEndpoint,
 } from "constants/dezswap";
+import { WCWallet } from "@interchain-kit/core";
 
 window.ResizeObserver = ResizeObserver;
 
 const queryClient = new QueryClient();
+const defaultOption = {
+  projectId: "bb7c346a9734b19a12c5d5828fa48eb8",
```

Comment:
Please use the app ID for reown org management: 80383e9eaec5f54a9cfa133d252c16b0